go/runtime/history/history_test.go

```diff
@@ -245,21 +265,22 @@ func TestWatchBlocks(t *testing.T) {
 	require.NoError(err, "New")
 	// No blocks should be received.
 	testWatchBlocks(t, history, 0)
-
 	// Commit a block.
-	err = history.Commit(&blk, roundResults, true)
+	err = history.Commit(&blk1, results1, false) // notify=false
+	require.NoError(err, "Commit should work")
+	// No blocks should be received since notify=false and !hasLocalStorage.
```

Comment:
I have already mentioned multiple times to use normal sentences instead of formulas or code! Comments should most of the time be written as text.